doc/api/sessions/sign_in_with_valid_password.json

```diff
@@ -16,23 +16,35 @@
       "description": "Password"
     }
   ],
+  "response_fields": [
+
+  ],
   "requests": [
     {
       "request_method": "POST",
       "request_path": "/users/sign_in",
       "request_body": "email=user1%40example.com&password=123456",
       "request_headers": {
-        "Accept": "application/json"
       },
       "request_query_parameters": {
       },
       "request_content_type": "application/x-www-form-urlencoded",
       "response_status": 201,
       "response_status_text": "Created",
-      "response_body": "{\"id\":1,\"authentication_token\":\"w3hSZvxJhansZ6bCss3y\",\"email\":\"user1@example.com\"}",
-      "response_headers": {"X-Frame-Options":"SAMEORIGIN","X-XSS-Protection":"1; mode=block","X-Content-Type-Options":"nosniff","X-UA-Compatible":"chrome=1","Content-Type":"application/json; charset=utf-8","ETag":"\"2b5b80af5b20c18e241900c6fae62462\"","Cache-Control":"max-age=0, private, must-revalidate","X-Request-Id":"8d421aac-9c05-43a7-a5f7-6daed1d5b02b","X-Runtime":"0.072063","Content-Length":"82"},
+      "response_body": "{\"user\":{\"id\":1,\"authentication_token\":\"wvEUWJZTjsevNTyPMGb1\",\"email\":\"user1@example.com\"}}",
+      "response_headers": {
+        "X-Frame-Options": "SAMEORIGIN",
+        "X-XSS-Protection": "1; mode=block",
+        "X-Content-Type-Options": "nosniff",
```

Comment:
The `X-Frame-Options` and `X-XSS-Protection` are mostly used by browser policies, should we get rid of them in the api version?